.env.example

```diff
@@ -0,0 +1,2 @@
+SUPABASE_URL=
+SUPABASE_ANON_KEY=
```

Comment:
Wouldn't it be just `SUPABASE_KEY`

Reply:
@0x4007 saw me specify `service_key` and said that we should always specify going forward even for anon key. I Can change it if you want

Reply:
If that is meant to be an anon key then this is fine.